docs/tools/webui-if.rst

```diff
@@ -66,6 +66,34 @@ use the WebUI to edit model metadata. To find out more, visit :ref:`organizing-m
 
 You can use the WebUI to view :ref:`historical-cluster-usage-data`.
 
+**************************
+ Viewing Cluster Topology
+**************************
+
+To view a resource pool's node and GPU distribution, as well as check which GPUs are currently in
+use, follow these steps:
+
+#. Navigate to Resource Pools.
+
+   From the left navigation pane, select **Cluster** to view **Resource Pools**.
+
+#. Check for the Topology section.
+
+   In the resource pool details page, you will find a **Topology** section visible if agents or
+   nodes are provisioned. If no agents or nodes are provisioned, the **Topology** section will not
+   be visible.
+
+#. View the Topology.
+
+   Under the **compute-pool** section, select the **Active slots** hyperlink to access the topology
```

Comment:
You don't need to select anything, whether you're in Active slots or Queued slots or any other view within the resource pool the topology will persist.

Reply:
having two ways to view the topology might be confusing to users: they can select a resource pool but there is also a hyperlink. i think the hyperlink is obvious while selecting a resource pool is less obvious.

Reply:
resolved by avoiding mention of the hyperlink